pkgs/servers/home-assistant/default.nix

```
@@ -88,50 +83,108 @@ in with py.pkgs; buildPythonApplication rec {
     owner = "home-assistant";
     repo = "core";
     rev = version;
-    sha256 = "07j54glcpa5ngkr0pwdg44f8gas3jz3nh653mr5sb5wg7xspgcr8";
+    sha256 = "182iw9dbz15zj47npvpxp531q3wdw81qnsy663d2bkvqlqszbiw5";
   };
 
+  patches = [
+    (fetchpatch {
+      #  Fix group tests when run in parallel, remove >= 0.117.0
+      url = "https://github.com/home-assistant/core/pull/41446.patch";
```

please use a stable uri (could be force pushed to)
```suggestion
      url = "https://github.com/home-assistant/core/commit/896f64d02b311b313fe04bff366be452381be49d.patch";
```

---

Indeed, thanks for pointing that out.